sno/diff_structs.py

```diff
@@ -17,6 +20,13 @@ def of(obj):
             return KeyValue(*obj)
         raise ValueError(f"Expected (key, value) tuple - got f{type(obj)}")
 
+    def get_lazy_value(self):
+        """Deltas can be created so that the values are only generated when needed."""
```

Comment:
Should this be memoized? If you call it twice will it call get_feature twice?

Reply:
Done